### ISSUE TITLE
Inclusion of 'polars' Python library instead of 'polars-lts-cpu' limits who can use this

### ISSUE THREAD

**Issue by mattdurant** [TracecatHQ/tracecat#246]
**Describe the bug**
The documentation does not call out in an obvious way that the use of the 'polars' library prevents anyone without AVX2 instructions from using this.  That means only AMD Zen 4 and newer or Intel Ice Lake and newer are able to run this using the pre-built Docker images from the registry.

**To Reproduce**
Run this on a machine with a CPU without AVX2 instructions.  The tracecat-worker and tracecat-api container will restart endlessly with entries in the log that the CPU does not support required instructions for the polars library.

**Expected behavior**
Give us an alternate image compiled with the polars-lts-cpu library instead of polars.

**Screenshots**
N/A

**Environment (please complete the following information):**

- OS: Debian 12
- CPU: Intel i7-4790K
- Browser version: N/A
- Docker version: 20.10.24
- Docker compose version: 2.18.1
- Are you using Docker desktop? No

**Additional context**
I did use the Dockerfile from the repo to make this change myself, and while the api container runs normally now, the worker container continues to restart repeatedly.

**Comment by topher-lo** [comment]
@mattdurant on it. Expect fix in the next hour 🫡

**Comment by mattdurant** [comment]
@topher-lo thanks for the fast turnaround, really looking forward to playing with this.  i didn't see any contribution guidelines, are you guys accepting PRs for more integrations/actions?

**Comment by topher-lo** [comment]
The new image should be out in ~15-30 minutes: https://github.com/TracecatHQ/tracecat/actions/runs/9980507504

> @topher-lo thanks for the fast turnaround, really looking forward to playing with this. i didn't see any contribution guidelines, are you guys accepting PRs for more integrations/actions?

@mattdurant we've got docs on building new integrations here: https://docs.tracecat.com/concepts/user-defined-functions. It's a really clean API (we designed it for ourselves after getting frustrated building integrations in other soar / workflow platforms...)

As for the contribution guidelines, we are working on it. Still a new project (first line of code in March), stabilizing our core features and user experience, but we have a really cool community strategy in mind. More on our approach to public integrations / playbooks library around DEFCON (2nd week August).
